packages/web-react/src/components/Stack/demo/StackBlocksWithCustomSpacing.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import DocsBox from '../../../../docs/DocsBox';
+import Stack from '../Stack';
+
+const StackBlocksWithCustomSpacing = () => (
+  <Stack elementType="ul" hasSpacing spacing="space-1200">
+    {[1, 2, 3].map((i) => (
+      <li key={i}>
```

Comment:
Please, do not use the array index as a key. Add a prefix for instance and it will be ok - `some-prefix-${i}`